Software/scan_key/AuthenticationSQL.py

```diff
@@ -8,16 +8,16 @@ def machineAuth(id, cursor):
     currMachineID = currMachineID.strip()
     currMachineID = b(currMachineID).decode('UTF-8')
 
-    cursor.execute("SELECT machineType FROM MACHINE WHERE machineID = " + currMachineID)
+    cursor.execute("SELECT authType FROM MACHINE WHERE PiHostname = " + currMachineID)
     
-    machineType = cursor.fetchone()
+    authType = cursor.fetchone()
 
-    cursor.execute("SELECT auth1, auth2, auth3 FROM USER WHERE t1String = " + id) #getting user w/ the id    #might need to modify
+    cursor.execute("SELECT " + authType + " FROM USER WHERE t1String = " + id) #getting user w/ the id    #might need to modify
```

Comment:
t1String is not a column in the `catsadmin`.`users` table on watt.cuMaker.space.

Will accept once this issue is resolved. Might be resolved by adjusting code, or simply adding the column to the table. I'll leave this up to the software team to review.